spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -72,6 +72,7 @@
  * @author Mike Eltsufin
  * @author Chengyuan Zhao
  * @author Daniel Zou
+ * @author Vinicius Carvalho
```

Comment:
It's hard to see what the changes are in the file since there is all this auto-formatting noise.
Can we hold the formatting fixes for another PR to make reviewing this one easier? Thanks!